buildbot/master/files/config/factories.py

```diff
@@ -215,6 +228,12 @@ def __init__(self, builder_name, environment):
                                  "etc/ci/buildbot_steps.yml")
         ])
 
+def make_pkill_command(target, is_windows):
```

Comment:
Similarly, I think this makes sense as a class method - please mind the duplication for now :)